components/map.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Mapsearch from './mapsearch.jsx';
+
+class Map extends React.Component {
+  constructor(props) {
+    super(props);
+    this.onSearch = this.onSearch.bind(this);
+  }
+
+  componentDidMount() {
+    const theMap = this.refs.map;
+    const mapOptions = {
+      center: {lat: 37.773972,
+      lng: -122.431297},
+      zoom: 12,
+      minZoom: 3
+    };
+
+    this.initializeMap();
+  }
+
+  initializeMap() {
+    var chicago = {lat: 41.850033, lng: -87.6500523};
+
+    var INITIAL_LOCATION = {
+      address: 'Chicago, Illinois',
+      position: {
+        lat: 37.773972,
+        lng: -122.431297
+      }
+    }
+
+    this.map = new google.maps.Map(this.refs.map, {
+      zoom: 4,
+      center: {lat: chicago.lat, lng: chicago.lng}
+    });
+
+    this.marker = new google.maps.Marker({
+
+      map: this.map,
+      position: {
+        lat: this.props.lat || INITIAL_LOCATION.position.lat,
```

Comment:
I think it's probably best to spell out lat and lng to latitude and longitude for the props. That's what I am expecting to come down through the store. Honestly, the only place I see 'lng' on Google Maps. Most places abbreviate it to long anyway.